webapp/src/containers/Rhs/index.tsx

```diff
@@ -90,62 +99,80 @@ const Rhs = (): JSX.Element => {
-                                        </Tooltip>
+                                    <div className='channel-link-icon'>
+                                        <img src={Constants.linkIconUrl}/>
                                     </div>
-                                </div>
-                                <Tooltip text={'Unlink'}>
-                                    <div className='channel-unlink-icon'>
-                                        <img
-                                            className='channel-unlink-icon__img'
-
-                                            // TODO: Update later
-                                            // eslint-disable-next-line no-alert
-                                            onClick={() => alert('Unlink chanel')}
-                                            src={Constants.channelUnlinkIconUrl}
-                                        />
+                                    <div className='link-data__ms-values'>
+                                        <img src={Constants.msteamsIconUrl}/>
+                                        <div className='link-data__body'>
+                                            <Tooltip text={link.msTeamsChannelName}>
+                                                <div className='link-data__channel-name'>{link.msTeamsChannelName}</div>
+                                            </Tooltip>
+                                            <Tooltip text={link.msTeamsTeamName}>
+                                                <div className='link-data__team-name'>{link.msTeamsTeamName}</div>
+                                            </Tooltip>
+                                        </div>
                                     </div>
-                                </Tooltip>
-                            </div>
-                        ))}
+                                    <Tooltip text={'Unlink'}>
+                                        <div className='channel-unlink-icon'>
+                                            <img
+                                                className='channel-unlink-icon__img'
+
+                                                // TODO: Update later
+                                                // eslint-disable-next-line no-alert
+                                                onClick={() => alert('Unlink chanel')}
+                                                src={Constants.channelUnlinkIconUrl}
+                                            />
+                                        </div>
+                                    </Tooltip>
+                                </div>
+                            ))}
+                        </InfiniteScroll>
+                        </div>
                     </div>
-                ) : (
+                )}
+                {totalLinkedChannels.length === 0 && !isLoading && (
```

Comment:
```suggestion
                {!totalLinkedChannels.length && !isLoading && (
```
Check if this works.